_includes/releases/v23.1/v23.1.0-alpha.4.md

```diff
@@ -0,0 +1,137 @@
+
+- Add columns p50, p90, p99, max and min latency for Statement table on SQL Activity page. [#97082][#97082]
+- Still show data on the console (with a warning) for Statement Insights when we reach a "max size exceed" error from the sql api. [#97153][#97153]
+- Still show data on the console (with a warning) for Transaction Insights when we reach a "max size exceed" error from the sql api. [#97277][#97277]
+- Still show data on the console (with a warning) for Schema Insights when we reach a "max size exceed" error from the sql api. [#97312][#97312]
+
+<h3 id="v23-1-0-alpha-4-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where the `AS OF SYSTEM TIME` clause was handled correctly in an implicit transaction that had multiple statements. [#97063][#97063]
+- This patch fixes asymmetric typing of `>` and `<` expressions which may cause erroring of expressions which are legal. [#97022][#97022]
+- This patch fixes possible internal errors in `SOME`, `ANY` and `ALL` expressions of the form: `expr > SOME(expr1, expr2, expr3...)` [#97022][#97022]
+- Fixed the syntax error for `SELECT ... QUERY` (without `AS`) statement. [#97041][#97041]
+- Decommissions that would violate constraints set on a subset of replicas for a range (e.g. `num_replicas = 3, <constraint>: 1`) will no longer be able to execute, respecting constraints during and after the decommission. [#94810][#94810]
+- Adjusted the size reported for `TIME`, `TIMETZ`, `TIMESTAMP`, `TIMESTAMPTZ`, and `DATE` types in the pgwire protocol. [#97145][#97145]
+- The ParameterStatus message is now only sent over the pgwire protocol if the value of the parameter changed. (The parameters that are sent this way are timezone, intervalstyle, datestyle, is_superuser, and application_name.) [#97145][#97145]
+- Users can now go to the next page of results when there are >20 active `stmts` or `txns` in the active exec pages. [#97122][#97122]
+- Since 22.1 when rangefeed enablement overrides in span configs were introduced, rangefeed requests that reached spans outside the range would not cause range cache invalidation due to the setting being checked first, thus requests could repeatedly hit the same incorrect range, causing errors until cache invalidation or node restart. This fix correctly checks that the span is within the range prior to checking the enablement settings, thus invalidating the cache when a request reaches an incorrect range and causing subsequent requests to successfully reach the correct range. [#97212][#97212]
+- A bug has been fixed that caused errors when creating multiple user-defined functions with the same name and different argument types in the same type family. For example, it was impossible to create both functions `f(i INT2)` and `f(INT4)`. [#96481][#96481]
+- The following spammy log message was removed: > lease [...] expired before being followed by lease [...]; foreground traffic may have been impacted [#97358][#97358]
```

Comment:
I would rewrite to avoid slang like "spammy" and to use code font for the literal message that is no longer logged.